schedule/yast/sle_15_sp3/create-hdd-y2-maintenance-with-addons.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: create-hdd-y2-maintenance-with-addons
```

Comment:
I would suggest to use `_` it is easier to select text in most editors and remove the `y2`. So something like `autoyast_create_hdd_mru_all_patterns` and `autoyast_create_hdd_mru_minimal_patterns`. wdyt?

Reply:
ahm ok it is only one, so just we should change `create_*` by `autoyast_create_hdd_maintenance.`